src/components/grid-view/GridViewWrapper.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import * as React from 'react';
+import GridView from './GridView';
+
+type Props = {
+    currentCollection: Collection,
+    height: number,
+    slotRenderer: (slotIndex: number) => React.Element<any>,
+    width: number,
+};
+
+const DEFAULT_COLUMN_COUNT = 5;
+const MAX_COLUMN_COUNT = 5;
+
+const ONE_COLUMN_BREAKPOINT = 700;
+const THREE_COLUMN_BREAKPOINT = 1400;
+
+const GridViewWrapper = ({ width, ...rest }: Props) => {
+    /**
+     * Determine the maximum number of columns based on the
+     * current width of the ContentExplorer.
+     *
+     * @private
+     * @return {number} - max number of columns
+     */
+    const getMaxColumns = (): number => {
```

Comment:
This probably doesn't need to be a separate function anymore; it can just be calculated directly and passed to `Math.min` below.